### DISCUSSION TITLE
RECENT_CRASH: 4175 daemons have recently crashed

### DISCUSSION THREAD

**Discussion by osaffer** [rook/rook#9917] [General]
Hi,

I used the command ceph crash prune 0 to clear the alert, unfortunately it did not work

RECENT_CRASH: 4175 daemons have recently crashed

The ceph cluster is totally clean, no more errors, but it stays in Warning because of that.

Any suggestions ?

Thanks

**Comment by satoru-takeuchi** [comment]
If it's not necessary to analyze the existing crash logs, `ceph crash archive-all` helps you. However, if the root cause of crashes still exist, new crashes will happen later and your cluster will be HEALTH_WARN again. Then I recommend you to take a look at crash logs, search the recent issues in the Ceph issue tracker, and create a new issue if necessary.

https://tracker.ceph.com/issues

**Reply by osaffer** [reply]
Thank you, it is exactly what I needed
